### PR TITLE
Update use-sso-in-event-based-activation.md

### PR DESCRIPTION
**Pull request by 1TheMuffinMan**
I probably spent 2 hours trying to debug this after adding it to my webserver before I googled the Office samples github repo and found out how it was supposed to be implemented :)

### CODE REVIEW

File: docs/outlook/use-sso-in-event-based-activation.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ The following example shows how to configure SSO or CORS for two add-ins (a main
 }
 ```
 
-Host the JSON file under a location named `.well-known` in the URI at the root of the origin. For example, if the origin is `https://addin.contoso.com:8000/`, then the well-known URI is `https://addin.contoso.com:8000/.well-known/microsoft-officeaddins-allowed.json`.
+Host the JSON file under a location named `.well-known` in the URI at the root of the origin. For example, if the origin is `https://addin.contoso.com:8000/`, then the well-known URI is `https://addin.contoso.com:8000/.well-known/microsoft-officeaddins-allowed.json`. For clarification, this file is to be hosted in your Office JavaScript add-in, not at the web server that you're attempting to make a CORS request to. For a more detailed example see [Office Add-in Samples](https://github.com/OfficeDev/Office-Add-in-samples/blob/51202dfea8c61c3c0f9fbd8a5f4075b2fd41ace3/Samples/auth/Outlook-Add-in-SSO-events/public/.well-known/microsoft-officeaddins-allowed.json#L5)
```
Comment (ElizabethSamuel-MSFT) [inline docs/outlook/use-sso-in-event-based-activation.md:31]:
```suggestion
Host the JSON file under a location named `.well-known` in the URI at the root of the origin. For example, if the origin is `https://addin.contoso.com:8000/`, then the well-known URI is `https://addin.contoso.com:8000/.well-known/microsoft-officeaddins-allowed.json`. For clarification, this file is to be hosted in your Office Web Add-in, not the web server that you're attempting to make a CORS request to. See the [Outlook-Add-in-SSO-events](https://github.com/OfficeDev/Office-Add-in-samples/blob/main/Samples/auth/Outlook-Add-in-SSO-events/public/.well-known/microsoft-officeaddins-allowed.json) sample for an example using the recommended location.
```

**Comment by 1TheMuffinMan** [comment]
@ElizabethSamuel-MSFT thanks for the suggestion! LGTM

**Comment by ElizabethSamuel-MSFT** [comment]
@1TheMuffinMan Sorry to hear that you faced this issue. We appreciate your taking the time to propose this update.

Currently, we're verifying the update with the feature crew. We'll merge if the crew confirms this change.

Thanks.

**Review by kwame-ansong — APPROVED** [review]

**Review by ElizabethSamuel-MSFT — APPROVED** [review]
@1TheMuffinMan Thanks again for making this update. This PR will be merged soon and the changes published later this week.

**Pull request merged**